test/EVMHost.cpp

```diff
@@ -266,7 +266,7 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 			bytes(begin(message.sender.bytes), end(message.sender.bytes)) +
 			bytes(begin(message.create2_salt.bytes), end(message.create2_salt.bytes)) +
 			keccak256(bytes(message.input_data, message.input_data + message.input_size)).asBytes()
-		));
+		), h160::AlignLeft);
```

Comment:
These will be fixed in #13533.

Reply:
What will be fixed? Doesn't this just preserve the original behavior by using the default here?

Reply:
This PR preserves current behaviour. The code will be "fixed" to be more accurate.

Reply:
Ah, ok.